MapExtension/SolARSample_Mapping_MapExtension_Mono/main.cpp

```diff
@@ -399,7 +399,8 @@ int main(int argc, char *argv[])
 				break;
 		}
 
-		// Save map
+		// Save map extension
+		mapper->bindTo<xpcf::IConfigurable>()->getProperty("directory")->setStringValue("mapExtension");
```

Comment:
Why not change this in the configuration file, like the others?

Reply:
Also, I know this didn't cause any issue as opposed to other mappings samples, but for the sake of consistency, maybe we could put "Fusion" maps (mapAB) either in `bin/` like we did for the others, or dedicate a special directory in data for all maps produced by samples (e.g. `data/output/maps/...`).
For now, just put everything in `bin/`.
Maybe later, we'd better add an argument to the sample (e.g. `--output-dir`) to change output dir programmatically like you did in the code above.

Reply:
Discussed offline:
- OK to put mapAB in bin/, I will do that.
- not done in configuration file for MapExtension, because directory serves as input as well as output, so here we change the value just before saving. Still thinks we should have an in and out directory in configuration, which could be set to be the same, but at least, we could set is via configuration file, or CLI, and not do this kind of hack.